app/experimenter/templates/experiments/section_analysis.html

```diff
@@ -7,4 +7,25 @@
 {% block section_content %}
   <p><strong>Data Science Owner: {{ experiment.analysis_owner }}</strong></p>
   {{ experiment.analysis|urlize|linebreaks }}
+
+  {{ experiment.SURVEY_REQUIRED_LABEL }}
+  <strong class="{% if experiment.survey %}text-danger{% endif %}">
+    {{ experiment.survey|yesno:"Yes,No" }}
+  </strong>
+
+  {% if experiment.survey %}
+  <br>
+    <strong>Survey URLs</strong>
```

Comment:
The spacing seems a bit strange here.

Reply:
also odd yes. I'm sorry for making you deal with that.